pipelines/skylab/smartseq2_single_nucleus_multisample/human_single_example.json

```diff
@@ -0,0 +1,18 @@
+{
+    "MultiSampleSmartSeq2.genome_ref_fasta": "gs://gcp-public-data--broad-references/hg38/v0/GRCh38.primary_assembly.genome.fa",
```

Comment:
these are in public references bucket already but the others aren't?

Reply:
That's correct. Will update the other file paths once they are moved.